challenges/objects-arrays.js

```diff
@@ -106,14 +184,20 @@ The zoos are concenred about animals with a lower population count. Find out whi
 
 */
 const lowerPopulation = [];
+
+let lowPop= zooAnimals.filter((population) => {
+  return zooAnimals .population <= 5;
+})
 console.log(lowerPopulation);
 
 /* Request 4: .reduce() 
 
 The zoos need to know their total animal population across the United States.  Find the total population from all the zoos using the .reduce() method.
 
 */
-const populationTotal = 0;
+const populationTotal = zooAnimals.reduce((total, population) => { 
```

Comment:
good job directly assigning the `populationTotal` value to the returned value from the `.reduce`